src/scss/components/_card.scss

```diff
@@ -45,7 +59,8 @@
   }
 
   &--highlighted {
-    .card__content {
+    [class*='-title'],
+    [class*='__title'] {
       color: get-color(rastaGray);
```

Comment:
Deep elements with a class that contains `title` in the name, will be affected with this css.